### PR TITLE
75 start gameover UI

### PR DESCRIPTION
**Pull request by dwarjie**
Implemented the Main page design into the game and also the gameover. Some stuff changed because I can't find a way to make them in css.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,27 +7,29 @@
...
+                <!-- <div id="instructions">
                     <h2 class="instructions-header">Instructions:</h2>
                     <p class="instructions-paragraph">
                         Lorem ipsum dolor sit amet consectetur adipisicing elit.
                         Magnam nulla explicabo magni sequi ipsam ad atque? Amet
                         vitae minus non optio consequatur iste. Molestiae dolore
                         quasi saepe, officiis necessitatibus modi!
                     </p>
-                </div>
+                </div> -->
```
Comment (kapaha) [inline src/index.html]: Need to remove all the comments

File: src/index.html [hunk 2]
```diff
@@ -7,27 +7,29 @@
         <!-- FONT LINK -->
         <link rel="preconnect" href="https://fonts.googleapis.com" />
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
Comment (kapaha) [inline src/index.html]: duplicate of above

**Review by kapaha — COMMENTED** [review]
looks great mark, good job, I just think it would look better in a fixed window instead of fullscreen as the game isn't responsive.

File: src/index.html (hunk 2 above)
Comment (tanselbay1) [reply src/index.html]: +1

**Review by tanselbay1 — CHANGES_REQUESTED** [review]
- Pause button overflow when clicked(Continue text overflow)
- Don't forget to check score board implementation which has Kai made.(When you do the merge conflict)
- Other than that looks good to me

File: src/index.html [hunk 3]
```diff
@@ -46,8 +48,15 @@ <h2 class="instructions-header">Instructions:</h2>
...
 
             <div id="game-over-page">
-                <h1 id="game-over-title" class="title">Game Over</h1>
-                <button class="button restart-button">Restart</button>
+                <img src="img/container.png" alt="Gameover" class="container" />
+                <div class="gameover-info">
+                    <h1 class="title">Game Over</h1>
+                    <h2 class="title-score">Score:</h2>
+                    <h2 id="game-over-score" class="score">10</h2>
+                    <h2 class="title-score">Wrong score</h2>
```
Comment (kapaha) [inline src/index.html]: Should be Wrong Answers:

File: src/index.html (hunk 1 above)
Comment (tanselbay1) [reply src/index.html]: We need a name and instructions of the game so these comments could stay here temporary

**Comment by kapaha** [comment]
> * Pause button overflow when clicked(Continue text overflow)
> * Don't forget to check score board implementation which has Kai made.(When you do the merge conflict)
> * Other than that looks good to me

**Review by tanselbay1 — APPROVED** [review]

**Pull request merged**